### ISSUE TITLE
Dependency Properties are declared as fields

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#2953]
## Current behavior

Many `DependencyProperty` declarations in Uno are `static readonly` fields, while UWP seems to use `static` read-only properties, for example in `Button`, Uno [has](https://github.com/unoplatform/uno/blob/9a05b80a6f2412cf23e2e4e114b082324cc8aaa6/src/Uno.UI/UI/Xaml/Controls/Button/Button.cs#L42):

```
public static readonly DependencyProperty FlyoutProperty = ...
```

But UWP [has](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.button.flyoutproperty):

```
public static DependencyProperty FlyoutProperty { get; } = ...
```

This is due to the fact that the "field" syntax was used in WPF and `propdp` in VS still generates this.

## Expected behavior

Class member kinds should match.

## How to reproduce it (as minimally and precisely as possible)

N/A

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by MartinZikmund** [comment]
@jeromelaban  This can be added to #2240

**Comment by jeromelaban** [comment]
This was fixed in 3.0
